internal/api/attestationconfig/client/client.go

```diff
@@ -19,51 +19,60 @@ import (
 	s3manager "github.com/aws/aws-sdk-go-v2/feature/s3/manager"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
 	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfig"
+	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfig/fetcher"
 	"github.com/edgelesssys/constellation/v2/internal/constants"
 	"github.com/edgelesssys/constellation/v2/internal/kms/storage"
 	"github.com/edgelesssys/constellation/v2/internal/sigstore"
 	"github.com/edgelesssys/constellation/v2/internal/staticupload"
 	"github.com/edgelesssys/constellation/v2/internal/variant"
 )
 
-// Client manages (modifies) the version information for the attestation variants.
-type Client struct {
+// Client is the client for the attestationconfig API.
+type Client interface {
+	UploadAzureSEVSNP(ctx context.Context, versions attestationconfig.AzureSEVSNPVersion, date time.Time) error
+	List(ctx context.Context, attestation variant.Variant) ([]string, error)
+	DeleteList(ctx context.Context, attestation variant.Variant) error
+	DeleteAzureSEVSNVersion(ctx context.Context, versionStr string) error
+}
```

Comment:
Unless there are a lot of places using this interface, prefer defining the interface at the place it is used, instead of the place where the implementation is.
See my previous comment for the `delete` command.